python/lsst/ap/verify/ap_verify.py

```diff
@@ -98,65 +98,75 @@ def __call__(self, value):
             raise argparse.ArgumentTypeError(self._message % value)
 
 
-def _get_output_dir(input_dir, output_arg, rerun_arg):
+def _getOutputDir(inputDir, outputArg, rerunArg):
     """Choose an output directory based on program arguments.
 
     Parameters
     ----------
-    input_dir: `str`
+    inputDir: `str`
         The root directory of the input dataset.
-    output_arg: `str`
-        The directory given using the `--output` command line argument.
-    rerun_arg: `str`
-        The subdirectory given using the `--rerun` command line argument. Must
-        be relative to `input_rerun`.
+    outputArg: `str`
+        The directory given using the `--output` command line argument. May
+        be None.
+    rerunArg: `str`
+        The subdirectory given using the `--rerun` command line argument.  May
+        be None, otherwise must be relative to `inputDir`.
 
     Raises
     ------
     `ValueError`:
-        Neither `output_arg` nor `rerun_arg` is None, or both are.
+        Neither `outputArg` nor `rerunArg` is None, or both are.
     """
-    if output_arg and rerun_arg:
+    if outputArg and rerunArg:
         raise ValueError('Cannot provide both --output and --rerun.')
-    if not output_arg and not rerun_arg:
+    if not outputArg and not rerunArg:
         raise ValueError('Must provide either --output or --rerun.')
-    if output_arg:
-        return output_arg
+    if outputArg:
+        return outputArg
     else:
-        return os.path.join(input_dir, "rerun", rerun_arg)
+        return os.path.join(inputDir, "rerun", rerunArg)
 
 
-def _measure_final_properties(metadata, metrics_job):
+def _measureFinalProperties(metadata, metricsJob):
     """Measure any metrics that apply to the final result of the AP pipeline,
     rather than to a particular processing stage.
 
     Parameters
     ----------
     metadata: `lsst.daf.base.PropertySet`
         The metadata produced by the AP pipeline.
-    metrics_job: `verify.Job`
+    metricsJob: `verify.Job`
```

Comment:
`lsst.verify.Job`

— needs the actual namespace to make the link.

Reply:
Huh. It looks like the dev guide contradicts itself here:
* https://developer.lsst.io/docs/py_docs.html#parameter-types says to use the full namespace
* https://developer.lsst.io/docs/py_docs.html#see-also says to start with the lowest common ancestor

Does the link generation actually work differently in these two contexts?

Reply:
Ah, I may need to update those docs. I guess we'll see if the link works when it's integrated into pipelines.lsst.io.

Reply:
Related question: should exceptions also be fully qualified? (There are a few that are not in `__builtins__`.)

Reply:
Yes, fully-qualified.